example/yarn.lock

```diff
@@ -5041,7 +5041,7 @@ lodash.uniq@^4.5.0:
   resolved "https://trendkite.jfrog.io/trendkite/api/npm/libs-frontend/lodash.uniq/-/lodash.uniq-4.5.0.tgz#d0225373aeb652adc1bc82e4945339a842754773"
   integrity sha1-0CJTc662Uq3BvILklFM5qEJ1R3M=
 
-"lodash@>=3.5 <5", lodash@^4.15.0, lodash@^4.17.11, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.2, lodash@^4.17.3, lodash@^4.17.4, lodash@^4.3.0:
+"lodash@>=3.5 <5", lodash@^4.15.0, lodash@^4.17.11, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.2, lodash@^4.17.3, lodash@^4.17.4, lodash@^4.3.0:
```

Comment:
Guess we missed a `yarn install` on a previous deploy?

Reply:
Maybe? This is the `example/yarn.lock` which doesn't get updated as often